doc/quickstart/gmt_quickstart.rst

```diff
@@ -1,7 +1,8 @@
 :Author: Hamish Bowman
 :Author: Massimo Di Stefano
 :Reviewer: Cameron Shorter
-:Version: osgeolive11.0
+:Reviewer: Felicity Brand (Google Season of Docs 2019)
+:Version: osgeolive12.0
```

Comment:
version 13 now, 14 next ?

Reply:
I am updating that version with the version I test against. I have 12 and 13, but for some reason my 13 is flaky sometimes and it crashes a bit.